Libraries/Experimental/SwipeableRow/SwipeableListView.js

```diff
@@ -101,6 +101,7 @@ class SwipeableListView extends React.Component {
 
   static defaultProps = {
     bounceFirstRowOnMount: false,
+    // $FlowFixMe(>=0.41.0)
```

Comment:
Looks like a bug in flow 0.41, it uses the type of `defaultProps` instead of `props`, it should be any function but ends up being `() => null`.